website/docs/reference/commands/dbt-environment.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "About dbt environment command"
+sidebar_label: "environment"
+id: dbt-environment
+---
+
+The `dbt environment` command enables you to interact with your dbt Cloud environment. The `dbt environment` command is useful for:
+
+- Viewing your local configuration details (account ID, active project ID, deployment environment, and more).
+- Viewing your dbt Cloud configuration details (account ID, environment ID, environment name, connection type, and more).
+
+This guide lists all the commands and options you can use with `dbt environment` in the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation). To use it, add a command after it like this: `dbt environment [command]`.
```

Comment:
i opted for plural here to make it easier for our future selves when more commands/options were added. adding